dojo/models.py

```diff
@@ -1687,6 +1689,17 @@ class Finding(models.Model):
                                         verbose_name="Number of occurences",
                                         help_text="Number of occurences in the source tool when several vulnerabilites were found and aggregated by the scanner.")
 
+    # this is useful for vulnerabilities on dependencies : helps answer the question "Did I add this vulnerability or was it discovered recently?"
+    publish_date = models.DateTimeField(null=True,
+                                         editable=False,
```

Comment:
should this be editable? Many parsers don't set this field and people may want to set it during triage, inspecting reference urls, etc.?

Reply:
good point, I hadn't noticed this field

Reply:
done. made the score editable too. However i'm thinking now that we should do something with the vector and the score: if the vector is given by the parser or the user, the score should be updated automatically. 
(If the score only is given, the vector may remain empty because the score only is not enough to know the full vector).